monkey/monkey_island/cc/ui/src/components/attack/techniques/T1090.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import '../../../styles/Collapse.scss'
+import ReactTable from "react-table";
+import { renderMachineFromSystemData, scanStatus } from "./Helpers"
+
+
+class T1090 extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  static getProxyColumns() {
+    return ([{
+      Header: "Proxies were used to communicate with:",
```

Comment:
I don't understand why you need 2 headers. It looks a bit weird...
The first header can be outside of the table

Reply:
True. Initially I had more columns, but after removing them I forgot to remove table header.